compiler/rustc_expand/src/mbe/macro_check.rs

```diff
@@ -335,6 +336,7 @@ fn check_occurrences(
             let name = MacroRulesNormalizedIdent::new(name);
             check_ops_is_prefix(sess, node_id, macros, binders, ops, span, name);
         }
+        TokenTree::MetaVarExpr(..) => {}
```

Comment:
@c410-f3r Yes, I believe `check_ops_is_prefix` would need to be modified to for meta-var-exprs. In particular, this function checks that meta-variables are defined when used, and that if they are used in (possibly nested) sequences, the Kleene operators and depth with which they use match those they are declared with. This would need to be modified slightly, I think as the meta-var-exprs can be used at any depth, iiuc...

Tracking this in another issue is fine, but it would be nice to fix this before the feature is stabilized.

Reply:
Thank you for the information, @mark-i-m! If possible, I will try to fix this situation as well as the other remaining features in a following PR as soon as possible.